Gruntfile.js

```diff
@@ -436,6 +439,15 @@ module.exports = function(grunt) {
         }
       });
 
+      if (config.appUpdateYML) {
+        var appUpdateYML = [dir, config.appUpdateYML].join('/');
```

Comment:
We have a lot of hard-coded `/` in the file, but we should start transitioning to `path.join()`